app/views/users/new.html.erb

```diff
@@ -0,0 +1,49 @@
+<%
+=begin%>
+<h1>User Registration</h1>
+<form action='/users' method='POST'> <br>
```

Comment:
This is missing `accept-charset="UTF-8"`. Should be `<form action='/users' method='POST' accept-charset="UTF-8">`